vault/external_tests/raftha/raft_ha_test.go

```diff
@@ -53,6 +54,20 @@ func TestRaft_HA_NewCluster(t *testing.T) {
 	})
 }
 
+// Test_RaftHA_Recover_Cluster test that we can recover data and re-boostrap a cluster
+// that was created with raft HA enabled but is not using raft as the storage backend.
+func Test_RaftHA_Recover_Cluster(t *testing.T) {
+	logger := logging.NewVaultLogger(hclog.Debug).Named(t.Name())
+	t.Run("file", func(t *testing.T) {
+		physBundle := teststorage.MakeFileBackend(t, logger)
+		testRaftHARecoverCluster(t, physBundle, logger)
+	})
+	t.Run("inmem", func(t *testing.T) {
+		physBundle := teststorage.MakeInmemBackend(t, logger)
+		testRaftHARecoverCluster(t, physBundle, logger)
+	})
```

Comment:
You mentioned this in a comment, and actually I think it would be great if we could have a raft test here too, since nothing should be different.

This should be something like:
```
	t.Run("raft", func(t *testing.T) {
		physBundle := teststorage.MakeRaftBackend(t, logger, 0, nil, &raft.ClusterAddrBridge{
		    clusterAddressByNodeID: make(map[string]string),
	         })
		testRaftHARecoverCluster(t, physBundle, logger)
	})
```

Reply:
I appreciate the attempt, and thanks for reporting back. This should be covered by `TestRaft_HA_ExistingCluster` in `raft_ha_test.go`, so I'm not too worried, but it would have been nice to have the symmetry. Definitely nothing that'll block the PR. Thanks! I'm going to approve this and try and get this merged.